File: scripts/unquote_oracle_sql.mjs
import fs from "fs";
const p2 = "g:/WorkAudit.CSharpOracle/Storage/Oracle/OracleBaselineInstaller.cs";
let s = fs.readFileSync(p2, "utf8");
s = s.replace(/"([a-zA-Z_][a-zA-Z0-9_]*)"/g, "$1");
fs.writeFileSync(p2, s);
console.log("ok");

File: scripts/unquote_oracle_sql.py
import re
path = r"g:/WorkAudit.CSharpOracle/Storage/Oracle/OracleBaselineInstaller.cs"
s = open(path, encoding="utf-8").read()
s = re.sub(r'"([a-zA-Z_][a-zA-Z0-9_]*)"', r"\1", s)
open(path, "w", encoding="utf-8").write(s)
print("done")
